analyze.py

```diff
@@ -118,17 +157,33 @@ def smb_analyze(self, responses):
             table.header(['Username', 'Password'])
             for x in successes:
                 table.add_row([x[0], x[1]])
+
+            # Calling notifications if specified
+            if self.notify == 'slack':
+                slack(self.webhook, self.host)
+            elif self.notify == 'teams':
+                teams(self.webhook, self.host)
+            elif self.notify == 'discord':
+                teams(self.webhook, self.host)
+
             print(table.draw())
+
+            # Returning true to indicate a successfully guessed credential
+            return True
+
+
         else:
             self.colors.color_print('[-] No successful SMB logins', self.colors.red)
 
```

Comment:
return false for the `check_sleep()` call in `spraycharles.py`

```suggestion
            return False
```